php/libraries/TimePoint.class.inc

```diff
@@ -148,6 +148,12 @@ class TimePoint implements \LORIS\StudyEntities\AccessibleResource
             );
             $title = $subprojectSettings['title'] ?? '';
             $this->_timePointInfo['SubprojectTitle'] = $title;
+
+            if ($this->_timePointInfo['ProjectID'] !== null) {
```

Comment:
I'm a bit confused by this part.

Reply:
What about it? The SQL query returned a nullable string. This is to convert it to a ProjectID type if it wasn't null.

Reply:
Now that I see that $this->_timePointInfo and $row store the same value, it makes sense, but it may be a little clearer to replace $row by $this->_timePointInfo

Reply:
I'm not sure I understand. The line is already `$this->_timePointInfo`?

Reply:
```
$this->_timePointInfo['ProjectID'] = new \ProjectID(
    $this->_timePointInfo['ProjectID']
);
```

Reply:
I changed it in the last commit.